schema/playground/shot.schema

```diff
@@ -0,0 +1,16 @@
+package playground;
+
+import "improbable/vector3.schema";
+import "from_gdk_packages/com.improbable.gdk.core/common.schema";
+
+type ShootRequest {
+    int32 damage = 1;
+    improbable.Vector3f start_point = 2;
```

Comment:
Why have both of these? Wouldn't a hit point make more sense?

Reply:
hit point and then check whether a game object is at that position? basically the idea is that the client only informs the server of the position of the entity and the direction of the shot and the server then checks what got hit

Reply:
That position and direction might not be consistent with what the other client sees though.
But the point of my question was why have both? What is the extra information used for?
If you had a hit point that would be used for playing animations etc.

Reply:
the information is there to do a raycast to identify which object was hit. 
or do you mean that we should change it to just sending which entity got hit directly and not do that on server-side?